### PR TITLE
AssetMeta Documentation

### PR DESCRIPTION
**Pull request by xitman78**
Refactored folder structure.
Inner components of AssetMeta have been removed from storybook and export.
Added AssetMeta Documentation.
Minor fixes and unit tests.

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-143.gearboxjs.preview.cogniteapp.com

File: src/components/AssetMeta/stories/hideTab.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## Alternate Pane
```
Comment (follan) [inline src/components/AssetMeta/stories/hideTab.md]: Should this title be Hide Tab?

File: src/components/AssetMeta/stories/selectedDocument.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+## Basic
```
Comment (follan) [inline src/components/AssetMeta/stories/selectedDocument.md]: Correct title?

File: src/components/AssetMeta/stories/hideTab.md (hunk 1 above)
Comment (xitman78) [reply src/components/AssetMeta/stories/hideTab.md]: fixed

File: src/components/AssetMeta/stories/selectedDocument.md (hunk 2 above)
Comment (xitman78) [reply src/components/AssetMeta/stories/selectedDocument.md]: fixed

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-143.gearboxjs.preview.cogniteapp.com

**Comment by follan** [comment]
LGTM

**Pull request merged**